spacy/util.py

```diff
@@ -996,9 +999,17 @@ def run_command(
     except FileNotFoundError:
         # Indicates the *command* wasn't found, it's an error before the command
         # is run.
+
+        if tool is None:
+            # On Windows we may be unable to get the executable name.
+            tool_check = ""
```

Comment:
To expand on why this is not possible, see the note on ambiguous file names in the [CreateProcess API docs](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-createprocessw#parameters). Basically, if the first argument of the command line is not explicitly quoted, the Windows API can treat the command as having unquoted spaces in the executable name.

> If you are using a long file name that contains a space, use quoted strings to indicate where the file name ends and the arguments begin; otherwise, the file name is ambiguous. For example, consider the string "c:\program files\sub dir\program name". This string can be interpreted in a number of ways. The system tries to interpret the possibilities in the following order:
> 
> c:\program.exe
> c:\program files\sub.exe
> c:\program files\sub dir\program.exe
> c:\program files\sub dir\program name.exe

We could split on a space and assume that usually works, but I think it's safest to just show the full command.